samza-core/src/test/scala/org/apache/samza/serializers/TestMetricsSnapshotSerde.scala

```diff
@@ -33,7 +34,7 @@ class TestMetricsSnapshotSerde {
   @Ignore
   @Test
   def testMetricsSerdeShouldSerializeAndDeserializeAMetric {
-    val header = new MetricsHeader("test", "testjobid", "task", "test", "version", "samzaversion", "host", 1L, 2L)
+    val header = new MetricsHeader("test", "testjobid", "task", "test", "test", "version", "samzaversion", "host", 1L, 2L)
```

Comment:
Minor comment: if we want to verify the result in the MetricsHeader in the unit test, it would be better to have a different string here for the execEnvironmentContainerId.